host_applications/linux/apps/raspicam/RaspiVid.c

```diff
@@ -2295,15 +2295,46 @@ static int wait_for_next_change(RASPIVID_STATE *state)
       char ch;
 
       if (state->verbose)
-         fprintf(stderr, "Press Enter to %s, X then ENTER to exit\n", state->bCapturing ? "pause" : "capture");
+         fprintf(stderr, "Press Enter to %s, X then ENTER to exit, [i,o,r] then ENTER to change zoom\n", state->bCapturing ? "pause" : "capture");
```

Comment:
Thanks.
No need to work around it, just wanted to ensure the text was correct. (Something must swallow the enter then, otherwise we'd toggle state, and then toggle straight back again as the enter was processed).